scripts/db_backup.sh

```diff
@@ -135,16 +135,25 @@ validate_remote_storage()
     fi
 
     # Container names cannot contain underscores or uppercase characters
-    CONTAINER_NAME="${DATABASE_TYPE}-backup"
+    if [[ "$DATABASE_TYPE" == "mongo" ]] || [[ "$DATABASE_TYPE" == "mysql" ]]
+    then
+        CONTAINER_NAME="${DATABASE_TYPE}-backup"
+    elif [ "$DATABASE_TYPE" == "trackinglogs" ] 
+    then
+        CONTAINER_NAME="trackingtest"
```

Comment:
Is this a fixed name? How does it change per environment? Also, you could simplify by using defaulting to CONTAINER_NAME="${DATABASE_TYPE}-backup" and conditionally updating for tracking logs